src/include/cache.i

```diff
@@ -133,93 +133,6 @@ __wt_cache_status(WT_SESSION_IMPL *session, int *evictp, int *dirtyp)
 }
 
 /*
- * __wt_eviction_check --
- *	Wake the eviction server if necessary.
- */
-static inline int
-__wt_eviction_check(WT_SESSION_IMPL *session, int *fullp, int wake)
```

Comment:
I think of cursor_enter and txn_begin as performance paths where it is likely that no eviction by application threads is required.  That's why this code was structured to inline the fast checks for those cases.

Can you please change that back?